### PR TITLE
Add analytics

### PR DESCRIPTION
**Pull request by dcravenus**

### CODE REVIEW

File: fs-album-slideshow.html [hunk 1]
```diff
@@ -1195,6 +1195,9 @@ <h4>[[i18n('slideshow_controls_slideshow_speed')]]</h4>
       }
     },
     toggleSocialMenu: function(e) {
+      this.fireMetrics({
+        link_name: 'Memories: Slideshow: Share'
```
Comment (jasonallen68) [inline fs-album-slideshow.html:1199]: The app name and page name are dynamically added to the link name, so you don't need to include it here.  Also, I would talk to Roger to make sure this is the link name he wants.  He's the one that will have to use it look up the analytics.

**Review by jasonallen68 — CHANGES_REQUESTED** [review]

File: fs-album-slideshow.html (hunk 1 above)
Comment (dcravenus) [reply fs-album-slideshow.html:1199]: Yeah, he told me to do 'Memories: Slideshow: Share' or if I must, 'Memories: Album: Slideshow: Share'

Comment (dcravenus) [reply fs-album-slideshow.html:1199]: So Memories and Slideshow are probably already included?

Comment (jasonallen68) [reply fs-album-slideshow.html:1199]: Not sure about  Slideshow.  The page portion of the link name comes from the .ejs page.

**Review by jasonallen68 — APPROVED** [review]

**Pull request merged**